SparkPost/support-docs
# WE-299 Fix cookie consent bar rendering bug

##What Changed
- Added new `ClientOnly` component that only renders children on the client and avoids SSR
- Wrapped the cookie consent component in `ClientOnly` to avoid the flash happening from server side rendering

##How to Test
- Open the deploy preview
- Dismiss the cookie consent bar
- Hard refresh the page and confirm the bar doesn't flicker or show up if it has been dismissed
